include/aos/common/tools/fs.hpp

```diff
@@ -440,7 +418,7 @@ class FS {
             return err;
         }
 
-        auto fd = open(fileName.CStr(), O_CREAT | O_WRONLY, perm);
+        auto fd = open(fileName.CStr(), O_CREAT | O_WRONLY);
```

Comment:
Why are permissions removed?

Reply:
Decided to do that because open is affected by umask for file creation. And it doesn't return an error if permissions of new file are different from requested.